bin/nuocd

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
```

Comment:
Maybe we should call this something else, since it actually is not specific to `nuocd`, e.g. `python-wrapper`.